test/functional/test_framework/util.py

```diff
@@ -250,90 +250,6 @@ def wait_for_bitcoind_start(process, url, i):
                 raise # unknown JSON RPC exception
         time.sleep(0.25)
 
-def initialize_chain(test_dir, num_nodes, cachedir, extra_args=None, redirect_stderr=False):
```

Comment:
The whole logic for `redirect_stderr` is lost now. Not sure if its still needed in `initialize_chain` however...I assume that all should be good as long as tests don't fail atm.